01.fizzbuzz/fizzbuzz.rb

```diff
@@ -0,0 +1,12 @@
+(1..20).each do |x|
+  if x%3 == 0 && x%5 == 0
```

Comment:
後々Rubocopというものを導入すると機械的に指摘してもらえることではありますが、基本的に演算子(`+`, `-`, `*`, `/`, `%`など)の前後にはスペースをあけた方が良いです。
```suggestion
  if x % 3 == 0 && x % 5 == 0
```

Reply:
`%` の前後にスペースを挿入いたしました。